tests/scripts/thread-cert/Cert_9_2_18_RollBackActiveTimestamp.py

```diff
@@ -170,6 +178,28 @@ def test(self):
         self.simulator.go(5)
         self.assertEqual(self.nodes[ROUTER2].get_state(), 'leader')
 
+    def verify(self, pv):
+        pkts = pv.pkts
+        pv.summary.show()
+
+        LEADER = pv.vars['LEADER']
+        COMMISSIONER = pv.vars['COMMISSIONER']
+        ROUTER_1 = pv.vars['ROUTER_1']
+        SED = pv.vars['SED']
+
+        # Step 1: Ensure the topology is formed correctly
+        pkts.filter_wpan_src64(ROUTER_1).filter_wpan_dst64(SED).filter_mle_cmd(MLE_CHILD_ID_RESPONSE).must_next()
+
+        # Step 3: Leader MUST send MGMT_ACTIVE_SET.rsp (Accept) to the Commissioner
+        pkts.filter_wpan_src64(LEADER).filter_coap_ack(MGMT_ACTIVE_SET_URI).must_next().must_verify(
```

Comment:
Better to check the destination address?

Reply:
checked with COMMISSIONER_RLOC